frontend/src/store/modules/dataset.js

```diff
@@ -234,6 +251,8 @@ const actions = {
         let upload = resource.upload;
         resource = JSON.parse(JSON.stringify(resource));
         resource = formatResourceBody(resource);
+        // eslint-disable-next-line
+        console.log("Resource: ", JSON.stringify(resource));
```

Comment:
Is this needed for logging or was just used for testing?

Reply:
removed this in commit https://github.com/bcgov/ckan-ui/pull/901/commits/82e162bfb456321bd11c72995f58b0bfe638b03a